src/main/scala/scalafix/sbt/BuildInfo.scala

```diff
@@ -5,13 +5,11 @@ import java.util.Properties
 object BuildInfo {
 
   def scalafixVersion: String =
-    props.getProperty("scalafixVersion", "0.9.3")
-  def scalafixStableVersion: String =
-    props.getProperty("scalafixStableVersion", "0.9.3")
+    props.getProperty("scalafixVersion", "0.9.6")
   def scalametaVersion: String =
-    props.getProperty("scalametaVersion", "4.1.0")
+    props.getProperty("scalametaVersion", "4.2.2")
```

Comment:
These hardcoded version numbers aren't really used since the property file should always be available. We might want to print a warning when the property file is missing saying that we're falling back to an old scalafix version 🤔

Reply:
I see, yeah we can do that!